compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -663,9 +664,13 @@ private void definePackageLevelVariables(DataInputStream dataInStream) throws IO
             // the symbol. Because, for the function pointers we directly read the param types
             // from the varType (i.e: from InvokableType), and assumes it can have only required
             // params.
-            varSymbol = new BInvokableSymbol(SymTag.VARIABLE, flags, names.fromString(varName),
+            BInvokableSymbol invokableSymbol = new BInvokableSymbol(SymTag.VARIABLE, flags, names.fromString(varName),
                                              this.env.pkgSymbol.pkgID, varType, enclScope.owner, symTable.builtinPos,
                                              toOrigin(origin));
+
+            invokableSymbol.kind = SymbolKind.FUNCTION;
+            invokableSymbol.retType = ((BInvokableType) invokableSymbol.type).retType;
```

Comment:
Shouldn't we set the param types too?

Reply:
https://github.com/ballerina-platform/ballerina-lang/blob/d67adbebf6c4c29a54a11ad219326ee003956a11/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java#L663-L666
according to this comment above this line, I didn't set them